mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/vault/attachements/AttachmentsFragment.kt

```diff
@@ -969,8 +969,8 @@ class AttachmentsFragment :
         binding.filterNameTv.text = when (sortType) {
             Sort.Type.DATE -> {
                 when (sortDirection) {
-                    Sort.Direction.ASC -> getString(R.string.Vault_SortDateAsc_SheetAction)
-                    Sort.Direction.DESC -> getString(R.string.Vault_SortDateDesc_SheetAction)
+                    Sort.Direction.ASC -> getString(R.string.Vault_SortDateDesc_SheetAction)
+                    Sort.Direction.DESC -> getString(R.string.Vault_SortDateAsc_SheetAction)
                 }
```

Comment:
You should also fix the design order if you're considering switching the strings.

Reply:
I already did the change of strings order in the popup :                                                                                             
     private fun handleSortSheet() {
        VaultSheetUtils.showVaultSortSheet(
            baseActivity.supportFragmentManager,
            getString(R.string.gallery_subheading_sort_by),
            getString(R.string.Vault_SortNameAsc_SheetAction),
            getString(R.string.Vault_SortNameDesc_SheetAction),
            getString(R.string.Vault_SortDateDesc_SheetAction),
            getString(R.string.Vault_SortDateAsc_SheetAction),
            sort = createVaultSortActions()
        )
    }